integration-tests/utils/workflow-utils.js

```diff
@@ -22,13 +22,18 @@ const launchWorkflowAndWaitForSuccess = async page => {
 
   // Long enough for the submission details to be Running or Failed.
   // Workflows might sit in "Submitted" for a long time if there is a large backlog of workflows in the environment
-  const isRunning = await Promise.race([
-    findInGrid(page, 'Failed', { timeout: 5 * 60 * 1000 }).then(() => false),
-    findInGrid(page, 'Running', { timeout: 5 * 60 * 1000 }).then(() => true)
+  const workflowStatus = await Promise.race([
+    findInGrid(page, 'Succeeded', { timeout: 5 * 60 * 1000 }).then(() => 'Succeeded'),
+    findInGrid(page, 'Failed', { timeout: 5 * 60 * 1000 }).then(() => 'Failed'),
+    findInGrid(page, 'Running', { timeout: 5 * 60 * 1000 }).then(() => 'Running')
```

Comment:
I admittedly do not fully understand how this test works, but it seems like we could completely ignore `Running`?

Testing the outcome of `Succeeded` transitively verifies the implementation detail that it was `Running` at some point in the past, and tests are best when they are about outcomes.

```suggestion
  const workflowStatus = await Promise.race([
    findInGrid(page, 'Succeeded', { timeout: 5 * 60 * 1000 }).then(() => 'Succeeded'),
    findInGrid(page, 'Failed', { timeout: 5 * 60 * 1000 }).then(() => 'Failed'),
```

Reply:
The way the test works now is that it waits up to 5 minutes for the workflow to start running (or fail).
https://github.com/DataBiosphere/terra-ui/blob/3ccf9fb37c47754b7bff174c71018950b57dd8fb/integration-tests/utils/workflow-utils.js#L23-L28

After the workflow starts running, the test waits up to 15 minutes for the workflow to succeed or fail.
https://github.com/DataBiosphere/terra-ui/blob/3ccf9fb37c47754b7bff174c71018950b57dd8fb/integration-tests/utils/workflow-utils.js#L35-L52

I don't know if that part of the test that asserts the workflow starts within 5 minutes is desirable or not.

Reply:
I guess what I'm saying is that if  the test has succeeded at the 15 minute mark, we don't care whether it was running at the 5 minute mark, because everything worked. But this is more of a thinking out loud exercise than an actual change request. Just trying to make things simpler and more resilient to variation.

Reply:
I'm going to go ahead and merge this as-is to attempt to resolve the current flakiness in tests. I agree with what you're saying though. Does anyone in the Batch or QA teams want to take on further updates to this test?